### PR TITLE
Extract as_array method to new class

### PR DESCRIPTION
**Pull request by harunyasar**
@joel-depiltech we can review and improve the changes if you want.

### CODE REVIEW

**Comment by harunyasar** [comment]
@joel-depiltech we can review and improve the changes if you want.

File: src/Transformer.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php namespace Odoo\Transformer;
```
Comment (joel-depiltech) [inline src/Transformer.php]: Maybe namespace must match with already set : Odoo\Transformer => Odoo\Client\Transformer

**Comment by joel-depiltech** [comment]
At this moment, I don't need some changes. You can merge if you feel ready

File: src/Transformer.php (hunk 1 above)
Comment (harunyasar) [inline src/Transformer.php]: I made the change ;)

File: src/OdooClient.php [hunk 2]
```diff
@@ -4,7 +4,7 @@
 use PhpXmlRpc\Value as xmlrpcval;
 use PhpXmlRpc\Request as xmlrpcmsg;
 use PhpXmlRpc\Response as xmlrpcresp;
-use Odoo\Transformer\Transformer;
+use Odoo\Client\Transformer\Transformer;
```
Comment (joel-depiltech) [inline src/OdooClient.php]: :-)

File: src/Transformer.php [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<?php namespace Odoo\Transformer;
+<?php namespace Odoo\Client\Transformer;
```
Comment (joel-depiltech) [inline src/Transformer.php]: :-)

**Comment by joel-depiltech** [comment]
Perfect ! Let's merge it !

**Pull request merged**